src/main/java/com/epam/ta/reportportal/ws/converter/builders/LaunchBuilder.java

```diff
@@ -101,16 +101,18 @@ public LaunchBuilder addAttributes(Set<ItemAttributeResource> attributes) {
 	}
 
 	public LaunchBuilder overwriteAttributes(Set<ItemAttributeResource> attributes) {
-		final HashSet<ItemAttribute> overwrittenAttributes = Sets.newHashSet(launch.getAttributes()
-				.stream()
-				.filter(ItemAttribute::isSystem)
-				.collect(Collectors.toSet()));
-		ofNullable(attributes).ifPresent(it -> it.stream().map(val -> {
-			ItemAttribute itemAttribute = FROM_RESOURCE.apply(val);
-			itemAttribute.setLaunch(launch);
-			return itemAttribute;
-		}).forEach(overwrittenAttributes::add));
-		launch.setAttributes(overwrittenAttributes);
+		if (attributes != null) {
+			final HashSet<ItemAttribute> overwrittenAttributes = Sets.newHashSet(launch.getAttributes()
```

Comment:
do we need to directly say that it is HashSet? can we use set that is created after stream collect and not to create a new one?